src/main/resources/templates/db_scripts/schema.json

```diff
@@ -64,6 +64,19 @@
           "targetTable": "po_line"
         }
       ]
+    },
+    {
+      "tableName": "order_invoice_relationship",
+      "generateId": true,
+      "fromModuleVersion": 6.0,
+      "withMetadata": true,
```

Comment:
The [order_invoice_relationship.json](https://github.com/folio-org/acq-models/blob/master/mod-orders-storage/schemas/order_invoice_relationship.json) schema does not include metadata so `withMetadata` should be `false`